src/pubsub/ua_pubsub.h

```diff
@@ -562,6 +562,14 @@ UA_StatusCode
 receiveBufferedNetworkMessage(UA_Server *server, UA_ReaderGroup *readerGroup,
                               UA_PubSubConnection *connection);
 
+void process_callback(UA_Server *server, UA_ReaderGroup *readerGroup,
+                      UA_PubSubConnection *connection, UA_ByteString *msg,
+                      UA_ByteString *topic);
```

Comment:
If it's only used for mqtt, than we should include "mqtt" or "broker" in the name.